README.md

```diff
@@ -1,217 +1,17 @@
 # hello-vault-go
 
-This is a sample application that demonstrates how to authenticate to and
-retrieve secrets from HashiCorp [Vault][vault].
+HashiCorp [Vault][https://www.vaultproject.io/] runnable examples in Go.
```

Comment:
Something grammatically strange about "Vault runnable" to me. How about "Runnable examples in Go of how to integrate an application with HashiCorp Vault.", or something like that?

Reply:
Ah, very good catch! I will change it